# Deskripsi Data

In [18]:
import pandas as pd

### Contoh Data

In [13]:
iris_df = pd.read_csv("IRIS.csv")

iris_df

,sepal_length,sepal_width,petal_length,petal_width,species
0,5.1,3.5,1.4,0.2,Iris-setosa
1,4.9,3.0,1.4,0.2,Iris-setosa
2,4.7,3.2,1.3,0.2,Iris-setosa
3,4.6,3.1,1.5,0.2,Iris-setosa
4,5.0,3.6,1.4,0.2,Iris-setosa
...,...,...,...,...,...
145,6.7,3.0,5.2,2.3,Iris-virginica
146,6.3,2.5,5.0,1.9,Iris-virginica
147,6.5,3.0,5.2,2.0,Iris-virginica
148,6.2,3.4,5.4,2.3,Iris-virginica


### Jumlah Baris, Kolom, Nama Atribut dan Tipe Data

In [ ]:
print("Baris: ", iris_df.shape[0])
print("Kolom: ", iris_df.shape[1])

Baris:  150
Kolom:  5


Data dari dataset iris tersebut memiliki jumlah baris dan kolom sebagai berikut:  

Baris: 150 baris  
Kolom: 5 kolom  

Data tersebut memiliki Atribut memiliki dengan nama sebagai berikut:  
1. sepal_lenght : Tipe Data merupakan Numerik (Continous) 
2. sepal_width : Tipe Data merupakan Numerik (Continous) 
3. petal_lenght : Tipe Data merupakan Numerik (Continous) 
4. petal_width : Tipe Data merupakan Numerik (Continous) 
5. species : Tipe Data Merupakan Kategorikal

